1566949583316	Sync.LogManager	DEBUG	Flushing file log
1566949583328	Sync.LogManager	DEBUG	Log cleanup threshold time: 1566085583328
1566949583348	Sync.LogManager	DEBUG	Done deleting files.
1566949981284	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1566949981284	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1566949981285	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1566949982284	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1566949987755	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1566949987755	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1566949987760	Sync.Service	DEBUG	User-Agent: Firefox/68.0.1 (Linux x86_64) FxSync/1.70.0.20190720144906.desktop
1566949987760	Sync.Service	INFO	Starting sync at 2019-08-28 01:53:07 in browser session dAGHbuoebQHi
1566949987761	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1566949987763	Sync.Status	INFO	Resetting Status.
1566949987763	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1566949987768	Sync.Resource	WARN	GET request to https://sync-592-us-west-2.sync.services.mozilla.com/1.5/120449125/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1566949987768	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1566949987768	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1566949987768	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1566949987768	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1566949987770	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
